addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/UniFiController.java

```diff
@@ -105,6 +107,54 @@ public void logout() throws UniFiException {
         executeRequest(req);
     }
 
+    public UniFiClient[] blockStation(UniFiClient client) throws UniFiException {
+        UniFiControllerRequest<UniFiClient[]> req = newRequest(UniFiClient[].class);
+
+        String siteName = "";
```

Comment:
sitename is a property of the handler therefor you should not try to find it here, but pass the sitename as parameter with this method. If due to some technical reason you really need to search for the sitename, it should be handled by the caller of this method.

Reply:
This is changed. Now given by a function.